[Bug] Compile of 0.65 fails on compiling undesired movie

#### Describe the issue

Compile failure

#### Steps to Reproduce
1. Get source code
``` bash
$ git show master
commit e954dd5ceeed709cc195b7b0334504b32c403677
...
```

2. Configure
``` bash
$ mkdir build
$ cd build
$ cmake -DWITH_MOVIES:BOOL=OFF \
      -DCMAKE_BUILD_TYPE=DEBUG \
      -DCMAKE_INSTALL_PREFIX=./installed \
      -DWITH_LUAJIT:BOOL=OFF \
      -DBUILD_ANIMVIEW:BOOL=ON \
      -DCMAKE_CXX_COMPILER:FILEPATH=/usr/bin/g++ \
      -DCMAKE_C_COMPILER:FILEPATH=/usr/bin/gcc \
      ..
   ```

3. Compile the program (ie `make`)

4. Observe failure to find include file.
``` text
/home/alberth/corsix-th/corsix_alberth_fork/CorsixTH/Src/th_movie.cpp:27:10: fatal error: libavutil/channel_layout.h: No such file or directory
   27 | #include <libavutil/channel_layout.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

Code there:
https://github.com/CorsixTH/CorsixTH/blob/master/CorsixTH/Src/th_movie.cpp#L27


Corsix pussycat died :cat2:

--- a/CorsixTH/Src/th_movie.cpp
+++ b/CorsixTH/Src/th_movie.cpp
@@ -24,8 +24,6 @@ SOFTWARE.
 
 #include "config.h"
 
-#include <libavutil/channel_layout.h>
-
 #include "lua_sdl.h"
 #if defined(CORSIX_TH_USE_FFMPEG) && defined(CORSIX_TH_USE_SDL_MIXER)
 
@@ -33,6 +31,7 @@ SOFTWARE.
 extern "C" {
 #include <libavcodec/avcodec.h>
 #include <libavutil/avutil.h>
+#include <libavutil/channel_layout.h>
 #include <libavutil/imgutils.h>
 #include <libavutil/mathematics.h>
 #include <libavutil/opt.h>

